code/modules/hydroponics/grown/melon.dm

```diff
@@ -39,6 +40,36 @@
 /obj/item/food/grown/watermelon/make_dryable()
 	return //No drying
 
+/obj/item/food/grown/watermelon/attackby(obj/item/I, mob/user, params)
+	if(!istype(I, /obj/item/kitchen/spoon))
+		return ..()
+
+	var/melon_pulp_count = 1
+	if(seed)
+		melon_pulp_count += round(seed.potency / 25)
+
+	user.balloon_alert(user, "scooped out [melon_pulp_count] pulp")
```

Comment:
"scoops out pulp"? It sounds a bit odd with just a number and extending the text to sound right would be wayyy too much text for a balloon alert.

Reply:
Why does it sound odd?